jina/orchestrate/flow/base.py

```diff
@@ -1497,6 +1499,14 @@ def _get_address_table(self, address_table):
                     attrs='underline',
                 )
             )
+            address_table.append(
```

Comment:
Add this according to `no_graphql_endpoint`

Reply:
`no_graphql_endpoint` then needs to be passed to the `Flow` and not only the `gateway` @JohannesMessner this should be seen in the `parser`

Reply:
That created some issues for @JoanFM and we decided to remove it. I wasn't able to replicate that problem, so I don't know.

Reply:
The problem was at which level the `args` was described. If you fix that, it would be good for me